fuzzers/005-tilegrid/add_tdb.py

```diff
@@ -76,34 +78,35 @@ def run(fn_in, fn_out, verbose=False):
     # FIXME: generate words from pitch
     int_frames, int_words = localutil.get_int_params()
     tdb_fns = [
-        ("iob/build/segbits_tilegrid.tdb", 42, 4),
-        ("ioi/build/segbits_tilegrid.tdb", 42, 4),
-        ("mmcm/build/segbits_tilegrid.tdb", 30, 101),
-        ("pll/build/segbits_tilegrid.tdb", 30, 26),
-        ("monitor/build/segbits_tilegrid.tdb", 30, 101),
-        ("bram/build/segbits_tilegrid.tdb", 28, 10),
-        ("bram_block/build/segbits_tilegrid.tdb", 128, 10),
-        ("clb/build/segbits_tilegrid.tdb", 36, 2),
-        ("cfg/build/segbits_tilegrid.tdb", 30, 101),
-        ("dsp/build/segbits_tilegrid.tdb", 28, 10),
-        ("clk_hrow/build/segbits_tilegrid.tdb", 30, 18),
-        ("clk_bufg/build/segbits_tilegrid.tdb", 30, 8),
-        ("hclk_cmt/build/segbits_tilegrid.tdb", 30, 10),
-        ("hclk_ioi/build/segbits_tilegrid.tdb", 42, 1),
-        ("clb_int/build/segbits_tilegrid.tdb", int_frames, int_words),
-        ("iob_int/build/segbits_tilegrid.tdb", int_frames, int_words),
-        ("bram_int/build/segbits_tilegrid.tdb", int_frames, int_words),
-        ("dsp_int/build/segbits_tilegrid.tdb", int_frames, int_words),
-        ("fifo_int/build/segbits_tilegrid.tdb", int_frames, int_words),
-        ("ps7_int/build/segbits_tilegrid.tdb", int_frames, int_words),
-        ("cfg_int/build/segbits_tilegrid.tdb", int_frames, int_words),
-        ("monitor_int/build/segbits_tilegrid.tdb", int_frames, int_words),
-        (
-            "orphan_int_column/build/segbits_tilegrid.tdb", int_frames,
-            int_words),
+        ("iob", 42, 4),
```

Comment:
I know you didn't add these in this CL, but it would be nice to have a comment about why some are hard coded and why some are not?

Reply:
All of the int frame TDB's are really the same constant, hence the variable `int_frames`/`int_words`.  All the others are ones offs and vary.